api/python/src/cellxgene_ontology_guide/supported_versions.py

```diff
@@ -0,0 +1,73 @@
+import functools
+import gzip
+import json
+import os
+from typing import Any, Dict, List, Optional
+
+from constants import DATA_ROOT, ONTOLOGY_FILENAME_SUFFIX, ONTOLOGY_INFO_FILENAME
+from semantic_version import Version
+
+from cellxgene_ontology_guide.entities import Ontology
+
+
+@functools.cache
+def load_ontology_file(file_name: str) -> Any:
+    """Load the ontology file from the data directory and return it as a dict."""
+    with gzip.open(os.path.join(DATA_ROOT, file_name), "rt") as f:
+        return json.load(f)
+
+
+def clear_ontology_file_cache() -> None:
+    """Clear the cache for the load_ontology_file function."""
+    load_ontology_file.cache_clear()
+
+
+def get_latest_schema_version(versions: List[str]) -> str:
+    """Given a list of schema versions, return the latest version.
+
+    :param versions: List[str] list of schema versions. Versions can be in the format "v5.0.0" or "5.0.0"
+    :return: str latest version with a "v" prefix
+    """
+
+    def _coerce(v: str) -> Version:
+        return Version.coerce(v[1:]) if v[0] == "v" else Version.coerce(v)
+
+    return "v" + str(sorted([_coerce(version) for version in versions])[-1])
+
+
+def load_supported_versions() -> Any:
+    """Load the ontology_info.json file and return it as a dict."""
+    with open(os.path.join(DATA_ROOT, ONTOLOGY_INFO_FILENAME)) as f:
+        return json.load(f)
+
+
+class CXGSchema:
+    """A class to represent the ontology information used by a cellxgene schema version."""
+
+    def __init__(self, version: Optional[str] = None):
```

Comment:
lets add a docstring explaining that version will default to latest, packaged version if not provided as input.